web/modules/custom/dpl_opening_hours/src/Controller/OpeningHoursEditorController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\dpl_opening_hours\Controller;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Defines OpeningHoursController class.
+ */
+class OpeningHoursEditorController extends ControllerBase {
+
+  /**
+   * Display the opening hours app.
+   *
+   * @return mixed[]
+   *   The app render array.
+   */
+  public function content() : array {
+    return [
+      '#theme' => 'dpl_react_app',
+      '#name' => 'opening-hours-editor',
+    ];
+  }
+
+  /**
+   * Check access for a specific node.
+   */
+  public function access(int $node) : AccessResult {
+    $nodeStorage = $this->entityTypeManager()->getStorage('node');
```

Comment:
Please use snake_case for variables in php :)
E.g. `$node_storage` and `$node_entity`